Resources/Prototypes/Entities/Objects/Consumable/Food/frozen.yml

```diff
@@ -0,0 +1,236 @@
+# Base
+
+- type: entity
+  parent: BaseItem
+  id: FoodFrozenBase
+  abstract: true
+  components:
+  - type: Food
+  - type: Sprite
+    sprite: Objects/Consumable/Food/frozen.rsi
+    netsync: false
+  - type: LoopingSound
+  - type: SolutionContainer
+    maxVol: 20 # For sprinkles or something? Idk.
+    contents:
+      reagents:
+      - ReagentId: Nutriment
+        Quantity: 10
+
+# Ice cream
+
+- type: entity
+  name: ice cream sandwich
+  parent: FoodFrozenBase
+  id: FoodFrozenSandwich
+  description: Portable Ice-cream in its own packaging.
+  components:
+  - type: Sprite
+    state: sandwich
+
+- type: entity
+  name: strawberry ice cream sandwich
+  parent: FoodFrozenBase
+  id: FoodFrozenSandwichStrawberry
+  description: Portable ice-cream in its own packaging of the strawberry variety.
+  components:
+  - type: Sprite
+    state: sandwich-strawberry
+
+- type: entity
+  name: space freezy
+  parent: FoodFrozenBase
+  id: FoodFrozenFreezy
+  description: The best icecream in space.
```

Comment:
Should probably decide on wether we want to use "ice-cream" or "icecream"

Reply:
Ice-cream.